### ISSUE TITLE
Clarify distinction between README and README.md

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/smallgrp#35]
It's not clear to me why there are both a README and a README.md file. Perhaps the intention is to merge the two? Then this should be recorded somewhere (e.g. this issue ;-).

Or perhaps they are meant to stay in parallel, which is also fine. But then I'd point from README.md to README. And I'd consider renaming README to e.g. `NOTES` or `INTERNALS` or whatever (and perhaps also convert it to Markdown?)

**Comment by alex-konovalov** [comment]
README is the old file by the initial authors. README.md new, created when work on the package was started. Agree that this requires some revision!

**Comment by fingolfin** [comment]
From the looks of it, I think a big part of `README` should actually be put into the package manual. The rest can be merged into `README.md`
